src/Smalot/PdfParser/RawData/RawDataParser.php

```diff
@@ -495,7 +495,9 @@ protected function getIndirectObject($pdfData, $xref, $objRef, $offset = 0, $dec
         $offset += strspn($pdfData, "\0\t\n\f\r ", $offset);
         // ignore leading zeros for object number
         $offset += strspn($pdfData, '0', $offset);
-        if (substr($pdfData, $offset, \strlen($objHeader)) !== $objHeader) {
+        // Newer pdf may use \n instead of whitespace
```

Comment:
What do you mean here with "Newer pdf"? Can you give me a reference to the specification?

Besides, could it be more than one \n at a time?

Reply:
Maybe I mistook the use cases, I was comparing a file in PDF 1.4 that throws MissingCatalogException with a file in PDF 1.7 that works. Sorry about that.
I do not know the great details of the PDF specifications, but I believe that the reference in the specifications is in [table 1 (7.2.2)](https://www.adobe.com/content/dam/acom/en/devnet/pdf/pdfs/PDF32000_2008.pdf)